### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by mikeubell**
I think these are a bit more complete.... Someone could add installing required sw on non-Mac systems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,26 @@ repository. Whereas django-calaccess-raw-data supports only state-level data, we
...
+
 ## Installation
 ```
 virtualenv env
 . env/bin/activate
 pip install -r requirements_dev.txt
+## create the database calaccess_raw
+mysql --user root 
+mysql> create database calaccess_raw;
+mysql> \q
```
Comment (adborden) [inline README.md:34]: If postgresql is a dependency, shouldn't we be using that as a DB instead of mysql here?

Comment (mikeubell) [inline README.md:34]: Where does it say that postgresql is a dependency?  The current install seems to use mysql.

Comment (adborden) [inline README.md:34]: Your PR adds postgresql as a dependency on [line 19](https://github.com/opencalifornia/disclosure-backend/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8R19). Is it actually required to have both mysql and postgres installed? (I probably already have both on my machine, so maybe I didn't notice).

Even though calaccess_raw supports both mysql and postgresql, I thought we were going with postgres. I could be wrong, but that's what I read from the [May meeting notes](https://docs.google.com/document/d/1uLf1HtW4HCvLB3l8E64vpk6b_P9DTxiFNaxo1Sy5t2U/edit#heading=h.ugpb38ewhkx)

Comment (seanius) [inline README.md:34]: only one is required, though the requirements_dev.txt lists both.  the aws
instance is currently using mysql, i forget why I set that up that way but
pretty sure there was a reason (previously i'd set up postgres but i think
during one of the in-person meetings there was a consensus for mysql.  no
horse in this race.)

Comment (mikeubell) [inline README.md:34]: I guess I should read what I write.  I'll try to reconstruct what I did.  I did some of I last week and the rest yesterday so I don't recall why I needed to install PostgreSQL.

**Comment by adborden** [comment]
:+1:

**Comment by mikeubell** [comment]
Update build instructions

**Comment by mikeubell** [comment]
looks like I don't have permission to merge this... someone should

**Pull request merged**